### ISSUE TITLE
Problem when using model from DIGITS

### ISSUE THREAD

**Issue by nicolasmetallo** [NVIDIA/gpu-rest-engine#11]
Hi,

I've trained my model under DIGITS 5 but when I try to make it work on GRE (following [these](https://github.com/NVIDIA/gpu-rest-engine/issues/4) steps) I keep receiving the same response, is it a bug because of the different versions of Caffe (DIGITS vs GRE)? Is there a workaround?
Thanks!!
Nicolas

[libprotobuf ERROR google/protobuf/text_format.cc:274] Error parsing text-format caffe.NetParameter: 53:18: Message type "caffe.BatchNormParameter" has no field named "scale_filler".
F0422 07:17:48.987526     1 upgrade_proto.cpp:88] Check failed: ReadProtoFromTextFile(param_file, param) Failed to parse NetParameter file: /models/money_reader/deploy.prototxt

**Comment by miraclewkf** [comment]
have you known the reason?

**Comment by flx42** [comment]
It could be because our fork `nvidia/caffe` has a different implementation of batch normalization, but I'm not an expert here. See https://github.com/NVIDIA/caffe/issues/276
